packages/manager/apps/web/client/app/domain/webhosting/order/steps/module/domain-webhosting-order-steps-module.html

```diff
@@ -0,0 +1,49 @@
+<oui-step-form
+    data-header="{{:: 'domain_webhosting_order_module_step_title' | translate }}"
+    data-description="{{:: 'domain_webhosting_order_module_step_subtitle' | translate }}"
+    data-editable="!$ctrl.stepper.loadingCheckout"
+    data-navigation="false"
+    data-on-focus="$ctrl.module = undefined">
+    <div
+      class="row d-md-flex mb-3"
+      data-ng-repeat="chunkModules in ::$ctrl.availableModules track by $index">
+         <div
+             class="col-md-4"
+             data-ng-repeat="module in ::chunkModules track by module.id">
+             <oui-select-picker
+                 class="h-100"
+                 data-name="{{:: module.planCode }}"
+                 data-label="{{:: 'domain_webhosting_order_module_step_' + module.planCode + '_label' | translate }}"
+                 data-model="$ctrl.module"
+                 data-values="[module]"
+                 data-variant="light"
+                 data-on-change="$ctrl.updateModule()">
+                 <oui-select-picker-description>
+                     <p data-translate="{{:: 'domain_webhosting_order_module_step_' + module.planCode + '_description' }}"></p>
+                     <span data-translate="domain_webhosting_order_module_step_recommended_offer"></span>
+                     <strong data-translate="{{:: 'domain_webhosting_order_hosting_step_' + module.recommendedOffer + '_label' }}"></strong>
+                 </oui-select-picker-description>
+                 <oui-select-picker-picture>
```

Comment:
Is there a reason that you use the `picture` transclude instead of the`picture` attribute ?
It sets an image: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-select-picker/src/select-picker.html#L19 :)